packages/cli/test/integration/shared/testDb.ts

```diff
@@ -35,21 +40,42 @@ export async function init() {
 
 	if (dbType === 'postgresdb') {
 		let bootstrapPostgres;
-		const bootstrapPostgresOptions = getBootstrapPostgresOptions();
+		const pgOptions = getBootstrapPostgresOptions();
 
 		try {
-			bootstrapPostgres = await createConnection(bootstrapPostgresOptions);
+			bootstrapPostgres = await createConnection(pgOptions);
 		} catch (error) {
-			const { username, password, host, port, schema } = bootstrapPostgresOptions;
-			console.error(
-				`ERROR: Failed to connect to Postgres default DB 'postgres'.\nPlease review your Postgres connection options:\n\thost: "${host}"\n\tusername: "${username}"\n\tpassword: "${password}"\n\tport: "${port}"\n\tschema: "${schema}"\nFix by setting correct values via environment variables:\n\texport DB_POSTGRESDB_HOST=value\n\texport DB_POSTGRESDB_USER=value\n\texport DB_POSTGRESDB_PASSWORD=value\n\texport DB_POSTGRESDB_PORT=value\n\texport DB_POSTGRESDB_SCHEMA=value`,
-			);
+			const pgConfig = getPostgresSchemaSection();
+
+			if (!pgConfig) throw new Error("Failed to find config schema section for 'postgresdb'");
+
+			const message = [
+				"ERROR: Failed to connect to Postgres default DB 'postgres'",
+				'Please review your Postgres connection options:',
+				`host: ${pgOptions.host} | port: ${pgOptions.port} | schema: ${pgOptions.schema} | username: ${pgOptions.username} | password: ${pgOptions.password}`,
+				'Fix by setting correct values via environment variables:',
+				`${pgConfig.host.env} | ${pgConfig.port.env} | ${pgConfig.schema.env} | ${pgConfig.user.env} | ${pgConfig.password.env}`,
+				'Otherwise, make sure your Postgres server is running.'
+			].join('\n');
+
+			console.error(message);
+
 			process.exit(1);
 		}
 
 		const testDbName = `pg_${randomString(6, 10)}_${Date.now()}_n8n_test`;
 		await bootstrapPostgres.query(`CREATE DATABASE ${testDbName};`);
 
+		try {
+			const schema = config.getEnv('database.postgresdb.schema');
+			await exec(`psql -d ${testDbName} -c "CREATE SCHEMA IF NOT EXISTS ${schema}";`);
+		} catch (error) {
+			if (error instanceof Error && error.message.includes('command not found')) {
+				console.error('psql command not found. Make sure psql is installed and added to your PATH.');
+			}
```

Comment:
hmmm so we now add `psql` as a dependency for the `test:postgres` command?

Reply:
Just like we are depending on the `mysql` command-line client:

https://github.com/n8n-io/n8n/blob/c38f6af4993cd695888ff18b3f95e0d900e65711/packages/cli/test/setup.ts#L22

Is there a way to avoid this?

Reply:
As discussed, `psql` ships with PG.